great_expectations/expectations/core/expect_multicolumn_sum_to_equal.py

```diff
@@ -103,6 +103,7 @@ def _prescriptive_renderer(
         runtime_configuration: Optional[dict] = None,
         **kwargs,
     ) -> None:
+        # TODO: Need for prescriptive renderer for Expectation if we want to render in DataDocs.
```

Comment:
Found this Expectation does not actually have any `renderers` implemented. Added `TODO`